<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceListPane</class>
 <widget class="QWidget" name="resource_list_pane_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>457</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="root_widget_layout_">
   <item>
    <widget class="QScrollArea" name="main_scroll_area_">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="main_content_">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>891</width>
        <height>437</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vertical_layout_stack_">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="pane_stack_">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="SnapshotStartPane" name="empty_page_"/>
         <widget class="QWidget" name="populated_page_">
          <layout class="QVBoxLayout" name="vertical_layout_" stretch="0,0,0,1,0,0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="GraphicsView" name="carousel_view_"/>
           </item>
           <item>
            <spacer name="vertical_spacer_">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="widget_" native="true">
             <layout class="QHBoxLayout" name="horizontal_layout_">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="ArrowIconComboBox" name="preferred_heap_combo_box_" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" stdset="0">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="ArrowIconComboBox" name="resource_usage_combo_box_">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item>
                <widget class="ScaledLabel" name="label_">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                 <string>Filter by size:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="RmvRangeSlider" name="size_slider_">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontal_spacer_">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>260</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="TextSearchWidget" name="search_box_"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="ScaledTableView" name="resource_table_view_" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="vertical_spacer_2_">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>5</height>
              </size>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="widget_2_" native="true">
             <layout class="QHBoxLayout" name="horizontal_layout_2_">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="ScaledLabel" name="label_2_">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Total resources:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="ScaledLabel" name="total_resources_label_">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>{0}</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="ScaledLabel" name="label_4_">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>|</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="ScaledLabel" name="label_5_">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Total size:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="ScaledLabel" name="total_size_label_">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>{0}</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontal_spacer_2_">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="ScaledPushButton" name="dump_resources_button_">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Dump resource table</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ArrowIconComboBox</class>
   <extends>QWidget</extends>
   <header>qt_common/custom_widgets/arrow_icon_combo_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RmvRangeSlider</class>
   <extends>QSlider</extends>
   <header>views/custom_widgets/rmv_range_slider.h</header>
  </customwidget>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>qt_common/custom_widgets/graphics_view.h</header>
  </customwidget>
  <customwidget>
   <class>ScaledLabel</class>
   <extends>QLabel</extends>
   <header>qt_common/custom_widgets/scaled_label.h</header>
  </customwidget>
  <customwidget>
   <class>ScaledPushButton</class>
   <extends>QPushButton</extends>
   <header>qt_common/custom_widgets/scaled_push_button.h</header>
  </customwidget>
  <customwidget>
   <class>ScaledTableView</class>
   <extends>QTableView</extends>
   <header>qt_common/custom_widgets/scaled_table_view.h</header>
  </customwidget>
  <customwidget>
   <class>TextSearchWidget</class>
   <extends>QLineEdit</extends>
   <header>qt_common/custom_widgets/text_search_widget.h</header>
  </customwidget>
  <customwidget>
   <class>SnapshotStartPane</class>
   <extends>QWidget</extends>
   <header>views/snapshot/snapshot_start_pane.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
